Makefile

```diff
@@ -30,7 +30,8 @@ images: $(patsubst %, build-%, $(TAGS))
 
 .PHONY: build-%
 build-%:
-	sed 's/%%ALPINE_TAG%%/$*/' Dockerfile.in >| Dockerfile
+	sed 's!%%ALPINE_TAG%%!$*!;s!%%ALPINE_REL%%!$(subst v,,$*)!' \
```

Comment:
This doesn't make sense to me.

`ALPINE_TAG` is `3.9`
`ALPINE_REL` is `3.9`

If you are looking to add `v` to the front of `ALPINE_TAG` I'd suggest `$(addprefix v,$*)`

Reply:
Yes that line slipped in and should be fixed. We need to fix the makefile to still support local build of images.